README.md

```diff
@@ -1,5 +1,5 @@
 # AWS Service Catalog Engine for Terraform Cloud
```

Comment:
Quick question: Should we update this header to reflect Enterprise support as well? 
```suggestion
# AWS Service Catalog Engine for Terraform Cloud / Enterprise
```

Reply:
I lean towards: "no", because I'd like to avoid confusion from folks who arrive at this page after searching for the "AWS Service Catalog Engine for Terraform Cloud" referenced in blog posts and AWS' documentation. Changing the name might lead folks to believe a DIFFERENT engine exists elsewhere.

Reply:
Fair! I'm totally fine with you keeping it as-is then. Thanks for the explanation 😊